chain/client/src/test_utils.rs

```diff
@@ -508,21 +539,28 @@ impl BlockStats {
     }
 }
 
+#[derive(Clone)]
+pub struct ActorHandlesForNetworkMessageMocking {
```

Comment:
nit: maybe ActorHandles is enough.

Reply:
I used this long name not for disambiguation but for clarity; ActorHandles might sound like it's not a testing-specific thing but rather some important construct. ActorHandlesForNetworkMessageMocking is definitely some silly struct just for testing :)

Reply:
Maybe ActorHandlesForTesting then?

Reply:
Sure!